bin/install

```diff
@@ -150,4 +154,15 @@ version_5_2x_or_greater() {
   fi
 }
 
+less_than_version_5_4x() {
+  version=$1
+  IFS='.' read -ra version_array <<<"$version"
+  major_minor_version="${version_array[0]}0${version_array[1]}"
+  if (("$major_minor_version" <= 504)); then
```

Comment:
This condition says less than or equal to 5.4.x, but the function name is `less_than_version_5_4x`. If the function name is correct this needs to become:

```bash
if (("$major_minor_version" < 504)); then
```